### ISSUE TITLE
Langes Laden im Personenmenü.

### ISSUE THREAD

**Issue by derryadrian** [OPUS4/application#389]
Um auf das Personenmenü zuzugreifen, dauert es bis zu 10 Sekunden, bis die erste Seite des Menüs angezeigt wird, die nur 50 Datensätze enthält. Es sieht so aus, als würde die Anwendung sehr komplexe Abfragen ausführen.
Nur in diesem Menü sind andere Menüs nicht.

![SharedScreenshot](https://user-images.githubusercontent.com/6037300/104575724-ef495180-5689-11eb-98bc-c4fa02ba5da3.jpg)

**Comment by j3nsch** [comment]
Ja, die Abfragen sind komplex. Momentan können wir daran nichts ändern, aber wenn wir zum weiteren Ausbau des Personen-Managements kommen, werden wir uns überlegen, wie wir die Abfragen effizienter handhaben können. Im Prinzip werden hier momentan ein Verhalten "simuliert", dass im Datenmodell konkret umgesetzt werden muss. Damit sollten die "typischen" Operationen wesentlich schneller werden.
